agent/app/agent.go

```diff
@@ -92,6 +93,7 @@ type ecsAgent struct {
 	vpc                   string
 	subnet                string
 	mac                   string
+	resource              resources.Resource
```

Comment:
This feels like the wrong naming to me, but I think we can leave it for now just to move this forward.

Reply:
`cgroup` isn't platform agnostic as it does not apply to `windows`